redirect/routes.php

```diff
@@ -8,9 +8,13 @@ class Route {
             'target' => 'http://tum.sexy/hunger'
         ],
         'mensabot' => [
-            'description' => 'TUMMensabot f&auml;r Telegram',
+            'description' => 'TUMMensabot für Telegram',
             'target' => 'https://t.me/TUMMensabot'
         ],
+            'mensabot2' => [
```

Comment:
Would you mind fixing the indentation?

Btw, you don't need to open a new PR when implementing this feedback 🙈  Just add another commit to your branch, I'm going to squash them anyway 😉

Reply:
The problem was that I used the quick-edit function of github I was not familiar with. That's why I thought "Why can't I just update my already existing pr to the current state of my repo? Why are my previous changes lost?". So sorry for failing so hard, the indentation seemed also to be correct in the online editing tool.